.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: Release X-Ray .NET SDK
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: The version to tag the release with, e.g., 1.2.0, 1.3.0
+        required: true
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout master branch
+        uses: actions/checkout@v2
+
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: 'v${{ github.event.inputs.version }}'
```

Comment:
We should add capital `V` here? looks like all the .NET tag follows V2.9.0, V2.8.0 and so on from [here](https://github.com/aws/aws-xray-sdk-dotnet/tags)